proto/one/operations/model/external/delphi_model_execution.proto

```diff
@@ -29,6 +29,8 @@ message DelphiModelExecution {
     repeated EnumDelphiModelMode modelModesEnabled = 10;
     //list of delphi configurationExecutions
     repeated DelphiConfigurationExecution configurationExecutions = 11;
+    //timezone offset in fraction hours of the Twin 
+    double timezoneOffsetHours = 12;
```

Comment:
This should probably be a nullable double here so we can tell the difference between a twin at GMT and one that does not have the setting.